custom/opm/reports.py

```diff
@@ -434,7 +434,7 @@ def _service_dates(self):
         for (owner_id, date), row in data.iteritems():
             if row['vhnd_available'] > 0:
                 for prop in VHND_PROPERTIES:
-                    if row[prop] == '1' or prop == 'vhnd_available':
+                    if row[prop] == 1 or prop == 'vhnd_available':
```

Comment:
We shouldn't really need ` or prop == 'vhnd_available':` since line 435 already implies `row['vhnd_available']` is `1`. Not sure why it's there.